frontend/src/_ui/Popover/index.jsx

```diff
@@ -14,16 +14,30 @@ const PopoverComponent = ({
   handleToggle,
   side = 'bottom',
   showArrow = false,
+  popoverContentHeight = '',
 }) => {
   const darkMode = localStorage.getItem('darkMode') === 'true';
-
+  const computeStyle = () => {
+    if (popoverContentHeight) {
+      return {
+        height: `${popoverContentHeight}vh`,
+        overflow: 'auto',
+      };
+    } else {
+      return {};
+    }
```

Comment:
```suggestion
    } 
      return {}
```

No need else statement as we're returning in if